app/util/jira/populate_db.sh

```diff
@@ -23,7 +23,7 @@ JIRA_DB_USER="postgres"
 JIRA_DB_PASS="Password1!"
 
 # Jira version variables
-SUPPORTED_JIRA_VERSIONS=(8.0.3 7.13.15 8.5.9)
+SUPPORTED_JIRA_VERSIONS=(8.13.0 8.5.9 8.0.3 7.13.15)
```

Comment:
do not change the order here, this order is used in by `force` flag to identify the latest version.